tools/pot/openvino/tools/pot/graph/passes.py

```diff
@@ -782,18 +829,24 @@ def create_fake_quantize_node(graph: Graph, name, data_type=np.float32):
     return fq
 
 
-def insert_fake_quantize(graph, node, ports=None, names=None):
+def insert_fake_quantize(graph, node, ports=None, names=None, fq_types=None, hw_config=None):
     blobs_as_inputs_nodes_type = ['Convolution', 'Deconvolution', 'MatMul']
 
     port_name = None
     if ports is not None and names is not None:
         port_name = dict(zip(ports, names))
 
+    fq_type = None
+    if fq_types is not None and ports is not None:
+        fq_type = dict(zip(ports, fq_types))
```

Comment:
Is this working? `fq_type` was defined as `None` just before the if statement.

Reply:
yes, the condition in Line 840 is checking for 'fq_types' not a 'fq_type'.